tests/common/dualtor/dual_tor_utils.py

```diff
@@ -1048,6 +1051,62 @@ def check_nexthops_balance(rand_selected_dut, ptfadapter, dst_server_addr,
                     pc))
 
 
+# verify nexthops are only sent to single active or standby mux
+def check_nexthops_single_downlink(rand_selected_dut, ptfadapter, dst_server_addr,
```

Comment:
I think this name is a little misleading since we're also checking the case where we expect all the packets to be sent to a single uplink. On that note, would it make more sense to break that case out into it's own helper function `check_nexthops_single_uplink`?